### ISSUE TITLE
Hardcoded text not translated

### ISSUE THREAD

**Issue by tamielbr** [funktechno/kanboard-plugin-wiki#13]
This issue is:

- [x] Bug report
- [ ] Feature request
- [ ] Improvement

### Actual behaviour
Some text is hardcoded and not in the localisation files.

### Expected behaviour
Text should translate to target language for the application.

### Screenshots
Example: 
![screen shot 2018-08-28 at 10 59 56](https://user-images.githubusercontent.com/3904166/44727938-9628f500-aab1-11e8-9a46-5bad75378698.png)

### Configuration

- Kanboard version: 1.2.40
- Database type and version: mysql/10.1.34-MariaDB
- PHP version: 7.1.19
- OS: mac
- Browser: firefox/safari

**Comment by lastlink** [comment]
Ok, I'll check on that, no big plans for updates for a month or so, welcome to make a pull request with the fix

**Comment by lastlink** [comment]
This was a missing translation not hardcoded as it was using the translate function 
```php
t('New wikipage')
```
in **create.php**. I only support Portuguese translations, but I also made some changes to spanish. Release 0.2.6 has a placeholder for each language in **locale** now for **New wikipage**, but it's not translated into every language. I think for your particular language I've now added the translation. #14 .
